darts/models/forecasting/ensemble_model.py

```diff
@@ -229,3 +257,26 @@ def find_max_lag_or_none(lag_id, aggregator) -> Optional[int]:
 
     def _is_probabilistic(self) -> bool:
         return all([model._is_probabilistic() for model in self.models])
+
+    def _full_past_covariates_support(self) -> bool:
```

Comment:
I think we should also adapt properties `supports_past_covariates` and `support_future_covariates` for ensemble models:

support X covariates if any of the underlying models supports X covariates.
WDYT?